pkg/minikube/cluster/cluster.go

```diff
@@ -340,6 +339,12 @@ func createHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 		return nil, fmt.Errorf("Error attempting to save store: %s", err)
 	}
 
+	// Create minishift-data.json
+	_, err = minishiftdata.Create(api.GetMachinesDir())
```

Comment:
!? Create again? You had a create in start.go as well, right - https://github.com/minishift/minishift/pull/519/files#r104312417